GenderPayGap.UnitTests/GenderPayGap.WebUI.Tests/GenderPayGap.WebUI.Tests.csproj

```diff
@@ -105,6 +105,10 @@
     <ProjectReference Include="..\..\GenderPayGap.WebUI\GenderPayGap.WebUI.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Folder Include="TestsCommon\Mocks" />
+  </ItemGroup>
+

```

Comment:
Do we need this?

Reply:
Probably not, I didn't add it intentionally